libpimeval/src/pimCmd.cpp

```diff
@@ -394,7 +394,7 @@ pimCmdFunc1::computeRegion(unsigned index)
         int64_t result = 0;
         if(!computeResult(signedOperand, m_cmdType, (int64_t)m_scalarValue, result, bitsPerElementSrc)) return false;
         objDest.setElement(elemIdx, result);
-      } else if (dataType != PIM_FP32) { // signed int.
+      } else if (dataType != PIM_FP32) { // unsigned int.
```

Comment:
Suggest defining an isUnsigned bool variable above and use it here. This avoids modifying this condition when introducing a new FP type.